packages/cli/src/lib/cmds/play.ts

```diff
@@ -232,10 +238,24 @@ async function setupGooglePlay(args: PlayArgs): Promise<GooglePlay> {
   return new GooglePlay(twaManifest.serviceAccountJsonFile);
 }
 
+export async function playPublish(parsedArgs: PlayArgs,
+    prompt: Prompt = new InquirerPrompt()): Promise<boolean> {
+  // TODO(@nohe427): Remove after experimental
+  if (!await prompt.promptConfirm(enUS.promptExperimentalFeature, false)) {
+    return true;
+  }
+  const googlePlay = await setupGooglePlay(parsedArgs);
+  const play = new Play(parsedArgs, googlePlay, prompt);
+  return await play.runPlayPublish();
+}
+
 export async function play(parsedArgs: PlayArgs,
```

Comment:
Would it be better to set this function to have an enum as an argument for if we want playX to run rather than having three methods? i.e., 
```
play(args, prompt, method) {
if method == publish {
  runPlayPublish()
}

Reply:
You can use something like

```ts
type PlayCommand = 'publish' | 'retain' | 'versionCheck';
```
then
```ts
function play(args: PlayArgs, prompt: Prompt = new InquirerPrompt(), command: PlayCommand) {
...
}
```

Reply:
Think I got this all resolved now. I changed the positioning of the args though since I think the prompt is an optional arg.